src/styles/index.scss

```diff
@@ -1,3 +1,131 @@
+html {
+  font-family: Roboto, Arial, Helvetica, sans-serif;
+}
+
 body {
   margin: 0;
 }
+
+.card {
+  box-sizing: border-box;
+  background-color: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  flex-direction: column;
+  width: 200px;
+  display: flex;
+}
+
+.image {
+  width: 160px;
+  height: 134px;
+  margin: 32px auto 0;
+  display: flex;
+}
+
+.product-name {
+  color: #060b35;
+  padding-inline: 16px;
+  margin-top: 40px;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  display: flex;
+}
+
+.product-code {
+  color: #616070;
+  margin-top: 4px;
+  padding-left: 16px;
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  display: flex;
+}
+
+.review {
+  justify-content: space-between;
+  align-items: flex-end;
+  margin-top: 16px;
+  padding-inline: 16px;
+  display: flex;
+}
+
+.review__stars {
+  align-items: center;
+  display: flex;
+}
+
+.review__stars-star {
+  background-image: url(/src/images/star.svg);
```

Comment:
The `background-image` URL should be relative to the `index.scss` file. It should start with `../images` instead of `/src/images` as per the task requirements.